include/api.php

```diff
@@ -3354,7 +3354,9 @@ function api_lists_statuses($type)
 	unset($_GET["screen_name"]);
 
 	$user_info = api_get_user($a);
-	// get last newtork messages
+	if (!x($_REQUEST, 'list_id')) {
```

Comment:
Please use `empty()` instead of `!x()`, `x()` will be deprecated in upcoming version.